rcl/test/rcl/test_arguments.cpp

```diff
@@ -14,33 +14,40 @@
 
 #include <gtest/gtest.h>
 #include <sstream>
+#include <string>
 #include <vector>
 
 #include "osrf_testing_tools_cpp/scope_exit.hpp"
+#include "rcpputils/filesystem_helper.hpp"
 
 #include "rcl/rcl.h"
 #include "rcl/arguments.h"
 #include "rcl/error_handling.h"
 
 #include "rcl_yaml_param_parser/parser.h"
 
+
 #ifdef RMW_IMPLEMENTATION
 # define CLASSNAME_(NAME, SUFFIX) NAME ## __ ## SUFFIX
 # define CLASSNAME(NAME, SUFFIX) CLASSNAME_(NAME, SUFFIX)
 #else
 # define CLASSNAME(NAME, SUFFIX) NAME
 #endif
 
+
```

Comment:
nit: extra new line

Reply:
Removed in https://github.com/ros2/rcl/pull/508/commits/c1f4c3ce1dc69cf0d29cf1c72873d582efa7a557